src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -1858,7 +1858,7 @@ public function isClean($attributes = null)
     }
 
     /**
-     * Determine if the model or any of the given attribute(s) have been modified.
+     * Determine if the model or any of the given attribute(s) were changed on save.
```

Comment:
Little unclear what this means. Do you mean these attributes were actually changed in the database on the last save?

Reply:
Thanks for reviewing, Taylor! I guess this is what the method does. My English is far from perfect so I will be happy if someone will do a better job. My goal is just to have PHPDoc comments which could minimize confusion between the methods